Yoast/javascript
# Add Dutch word form support

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds word form support for Dutch.
* [Yoast SEO Premium] Adds word form support for Dutch.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Checkout the `feature/Dutch-morphology` on `javascript`
* Checkout the `feature/Dutch-morphology` on `YoastSEO.js-premium-configuration`
* Link this branch in your premium plugin using `yarn link-monorepo`
* Build the premium plugin as usual
* Check that the plugin recognizes different forms of keyword in the text, title, slug and meta description.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  * Keyphrase recognition

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-62
